### PR TITLE
Removing eosq go server

### PR DESCRIPTION
**Pull request by billettc**

### CODE REVIEW

File: mdl/v1/block.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package mdl
+
+import (
+	"time"
+)
+
+type Block struct {
```
Comment (abourget) [inline mdl/v1/block.go]: rename to BlockSummary.. it isn't a `Block` by any stretch of the imagination.  a `Block` from a top-level package should represent the best we know of a block.. but that would be in `eos-go` anyway.

File: mdl/v1/block.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	ID               string       `json:"id"`
+	Irreversible     bool         `json:"irreversible"`
+	Header           *BlockHeader `json:"header"`
+	TransactionCount int          `json:"transaction_count"`
+}
+
+type BlockHeader struct {
```
Comment (abourget) [inline mdl/v1/block.go]: We have the one in `eos-go`.. please use that one.. if there's anything different, then create a wrapper with a different name. Hopefully we don't need to do that.

File: mdl/v1/transaction.go [hunk 3]
```diff
@@ -88,3 +88,8 @@ type ActionRef struct {
 	TrxID       string `json:"trx_id"`
 	ActionIndex int    `json:"action_index"`
 }
+
+type TransactionList struct {
+	NextCursor   string                  `json:"next_cursor"`
```
Comment (abourget) [inline mdl/v1/transaction.go]: Check for consistency with `eosws`.. we use `cursor` over there I think. Make it the same.

**Review by abourget — CHANGES_REQUESTED** [review]

**Comment by abourget** [comment]
Est-ce qu'on a besoin d'ajouter tout ça à la lib? Tant qu'on rends pas publique les endpoints Blocks, on n'a pas vraiment besoin de ça ici, right?

**Comment by maoueh** [comment]
Closing for now, all comments were addressed and work has been merged in `dev` branch.

Going to open a PR to merge `dev` into `master` and see to tag release.

**Pull request closed without merging**